python/tk_desktop/desktop_window.py

```diff
@@ -505,9 +507,11 @@ def show_update_project_config(self):
         self.update_project_config_widget.show()
         self.project_overlay.hide()
 
+    def _get_current_user(self):
+        return login.get_current_user(sgtk.platform.current_engine().tank)
```

Comment:
This should ideally use the official, public, API method `sgtk.util.get_current_user()` - see the docs here - https://toolkit.shotgunsoftware.com/entries/23808858#Utility%20Functions